data/campaigns/The_Deceivers_Gambit/units/Orcish_Sorceress.cfg

```diff
@@ -21,7 +21,7 @@
     advances_to=null {AMLA_DEFAULT}
     cost=54
     usage=archer
-    description= _ "Orcs are sometimes capable as living as long as humans, but very few individuals ever fully realize that potential, as the average orc only sees two or three decades before meeting their end either in battle. Only the Shamans, isolated from the daily strife of the major orcish clans, commonly live to reach old age.
+    description= _ "Orcs are sometimes capable as living as long as humans, but very few individuals ever fully realize that potential, as the average orc only sees two or three decades before meeting their end either in war or by the hand of one of their kin. Only the Shamans, isolated from the daily strife of the major orcish clans, commonly live to reach old age.
```

Comment:
Also, the first "as" should be "of" instead.

Orcs are sometimes capable ~as~ **of** living as long as humans